app/controllers/admin/users_controller.rb

```diff
@@ -89,6 +89,7 @@ def user_params
     params.require(:user).permit(
       :full_name,
       :phone_number,
+      :teleconsultation_phone_number,
```

Comment:
Should we model this as an isd code and number, like we do for the existing telemed numbers? I think we found that works, and gives us flexibility as we don't hard-couple the phone number to the country.

Reply:
Added in 9e00e02. Small nuance:
- Currently `teleconsultation_phone_numbers` are sent as `+91222222222` since whatsapp expects `912222222222`.
- User `phone_numbers` are sent as `22222222222` because phone number auths don't have a country code.

We will prepend the isd code before sending it in the `user` payload so that `teleconsultation_phone_number` will always be in the `+91` format. This does mean that the user payload will have `phone_number` and `teleconsultation_phone_number` in different formats. I'll run this by mobile devs, not expecting this to cause any issues. Just something to be aware of.

![image](https://user-images.githubusercontent.com/16774200/91307225-52197980-e7cb-11ea-99b0-495cee8c1242.png)

Ran the UI change by claudio ✅

Reply:
Coolbeans. Yeah it's a little weird because the app doesn't care about the teleconsult number exposed in this payload, it only cares about the "MOs in a facility" endpoint, where we can consistently deliver phone numbers with ISD codes, like we do now.

Don't worry _too_ much about getting it perfect, I think there's gonna be some inconsistency no matter what we decide - either between these numbers and existing telemed numbers, or between these telemed numbers and user phone auths.